Sources/UIKitExtensionsSorceryGeneration/UIActionSheet+PropertyConfigurationSet.swift

```diff
@@ -0,0 +1,36 @@
+// Generated using Sourcery 0.10.1 — https://github.com/krzysztofzablocki/Sourcery
+// DO NOT EDIT
+
+import UIKit
+public extension ConfigurationSet where Base: UIActionSheet {
+    func delegate(_ newValue: UIActionSheetDelegate?) -> ConfigurationSet<Base> {
+        return set{ (UIActionSheet: UIActionSheet) in
```

Comment:
the parameter names should be written in lowerCamelCase by convention. But all generated parameter names start with upperCases letters. This make it look like calling Static methods/properties.

Maybe it would be better to call them "configurable" or similar.